### PR TITLE
Add support for openj9 personal repo and branch

### PR DESCRIPTION
**Pull request by llxia**
Issue: #415

Signed-off-by: lanxia <lan_xia@ca.ibm.com>

### CODE REVIEW

**Comment by llxia** [comment]
Tests:
- jdk_math with default (openj9 master): https://ci.adoptopenjdk.net/view/work%20in%20progress/job/TestBuild_Sandbox/388/parameters/
- cmdLineTester_classesdbgddrext with my personal branch: https://ci.adoptopenjdk.net/view/work%20in%20progress/job/TestBuild_Sandbox/389/parameters/

**Comment by smlambert** [comment]
I updated the wiki page with Grinder and these changes: https://github.com/AdoptOpenJDK/openjdk-tests/wiki/How-to-Run-a-Personal-Test-Build-on-Jenkins

File: buildenv/jenkins/Jenkinsfile_personal [hunk 1]
```diff
@@ -2,8 +2,8 @@
 pipeline {
     agent {label "${getPlatformAndLabel(SPEC)[1]}"}
 	parameters {
-		string (defaultValue: "AdoptOpenJDK", description: 'personal repo from AdoptOpenJDK/openjdk-tests', name: 'personalRepo')
-		string (defaultValue: "master", description: 'personal branch', name: 'personalBranch')
+		string (defaultValue: "https://github.com/AdoptOpenJDK/openjdk-tests.git", description: 'personal repo from AdoptOpenJDK/openjdk-tests', name: 'ADOPTOPENJDK_REPO')
+		string (defaultValue: "master", description: 'personal AdoptOpenJDK branch', name: 'ADOPTOPENJDK_BRANCH')
 		choice (choices: 'linux_x86-64\nlinux_x86-64_cmprssptrs\nmac_x86-64\nlinux_390-64\nlinux_390-64_cmprssptrs\nlinux_ppc-64_le\nlinux_ppc-64_cmprssptrs_le\nlinux_arm\nwin_x86-64\nwin_x86-64_cmprssptrs\naix_ppc-64\naix_ppc-64_cmprssptrs', description: 'SPEC?', name: 'SPEC')
 		choice (choices: 'SE80\nSE90\nSE100', description: 'What is JAVA_VERSION?', name: 'JAVA_VERSION')
```
Comment (smlambert) [inline buildenv/jenkins/Jenkinsfile_personal]: we may eventually want to add the other values in the choice list... amber, valhalla, panama

**Comment by smlambert** [comment]
Related to @sophia-guo PR #414 - you can get rid of the ci.project.openj9 part of the label, in that way this Jenkinsfile will work as is at the 4 Jenkins servers they are currently being used at (or you can wait for her PR to deliver that change).

**Comment by sophia-guo** [comment]
Thanks @ShelleyLambert! Will update when this is done.

**Pull request merged**